<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MAUI</class>
 <widget class="QWidget" name="MAUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>87</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Moving average</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="maLabel">
     <property name="text">
      <string>MA type</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="maComboBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="ssLabel">
     <property name="text">
      <string>Slow smoothing</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="ssSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="fsLabel">
     <property name="text">
      <string>Fast smoothing</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="fsSpinBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>maLabel</zorder>
  <zorder>maComboBox</zorder>
  <zorder>ssLabel</zorder>
  <zorder>ssSpinBox</zorder>
  <zorder>fsLabel</zorder>
  <zorder>fsSpinBox</zorder>
  <zorder>titleLabel</zorder>
  <zorder>line</zorder>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>setValues()</slot>
 </slots>
</ui>
